specification/storage/resource-manager/Microsoft.Storage/preview/2020-08-01-preview/examples/BlobContainersClearLegalHold.json

```diff
@@ -0,0 +1,25 @@
+{
+  "parameters": {
+    "subscriptionId": "{subscription-id}",
+    "resourceGroupName": "res4303",
+    "accountName": "sto7280",
+    "containerName": "container8723",
+    "api-version": "2019-06-01",
```

Comment:
The API version in samples also need change

Reply:
Thanks! fixed.